.gitignore

```diff
@@ -3,3 +3,5 @@ pkg/
 .idea
 .DS_Store
 .vscode/settings.json
+.devenv
+.direnv
```

Comment:
You should probably commit these in your `~/.config/git/ignore` or `~/.gitignore` (alongside with .envrc), your global ignore file, rather than a project-level ignore file.